tests/action-menu/copy-link.spec.js

```diff
@@ -45,7 +45,7 @@ mainTest.afterEach(async () => {
 
 mainTest.describe(() => {
   mainTest(
-    qase(2036, 'Share link of two Boards to a user from your team'),
+    qase([2036], 'Share link of two Boards to a user from your team'),
```

Comment:
It's nice to leave it prepared to accept not just a single test, but a potential list. Less research to do in that case. :+1: